jsonschema-validation.xml

```diff
@@ -575,21 +559,23 @@
                     validates against the provided schema.
                     Note the property name that the schema is testing will always be a string.
                 </t>
+                <t>
+                    A missing keyword has the same behavior as an empty schema.
```

Comment:
If empty schema vs `true` is the only concern, I endorse doing that in a follow-up PR.  We should check for it in more places than just these lines, and that will be trivially easy to check up on on its own.